src/adapter.js

```diff
@@ -192,10 +317,20 @@ export function parse({minim, source}, done) {
         }
 
         // Currently, default responses are not supported in API Description format
-        const relevantResponses = _.omit(methodValue.responses, 'default');
+        // TODO: [Annotation] Add warning about not showing the default!
+        const relevantResponses = _.chain(methodValue.responses)
+          .omit('default')
+          .omit(isExtension)
+          .value();
 
         if (_.keys(relevantResponses).length === 0) {
-          createTransaction(minim, transition, method);
+          if (bodyParameters.length) {
+            // Create an empty successful response so that the request/response
+            // pair gets properly generated.
+            relevantResponses[200] = {};
```

Comment:
The bug this is trying to solve is that before this change if there were request parameters defined but no responses defined, then we would *not* generate a request refract object and the input parameters would be completely ignored.

We can either make my change above or we can refactor the other code to only create a request rather than a request and an empty response. Given that we already create blank requests all over the place I think my solution above makes sense, and I'm struggling to understand cases where this would be an issue in other tooling.